### DISCUSSION TITLE
Can Bus

### DISCUSSION THREAD

**Discussion by orbitcoms** [Mair/esp32-course#107] [Course topic requests]
Would be interested in CanBus video

**Comment by mikebeepboop** [comment]
Have you seen this? There is a Two Wire Automotive Interface (TWAI), it has some example code:
https://docs.espressif.com/projects/esp-idf/en/latest/esp32/api-reference/peripherals/twai.html

Also some great examples here:
https://github.com/espressif/esp-idf/tree/master/examples/peripherals/twai/twai_network
